ONSdigital/sdx-receipt-rrm
# Fix logger bindings

## What? and Why?
There was a bug where every submission after the first would have the previous submissions tx_id, ru_ref and user_id in the logs between the start and after the decryption. After the decryption, the correct one became bound. This fixes that issue.

Additionally updates urllib3 to fix a security vulnerability.

## How to test
- Check out master
- Either add a log line before the decrypt, or change the the SDX_RECEIPT_RRM_SECRET.  Either need to be done because there isn't any logging in the master branch before the correct binding is done unless an error occurs.
- Put through 2 submissions.  You'll notice on the second submission there are references to the first submissions tx_id, user_id and case_id in the first few lines (around the time it decrypts it)
- Check this branch out
- Put through 2 more submissions and notice that the incorrect data is not there anymore.  In this branch, I've added some more logging around the decryption and validation.  The absence of previous data in these lines is enough to call this bugfix correct.

## Checklist
